package.json

```diff
@@ -16,19 +18,36 @@
     "meta-function-check": "./dist/src/bin/validate-repo.js",
     "meta-package-check": "./dist/src/bin/validate-package.js"
   },
+  "mocha": {
+    "require": [
+      "ts-node/register"
+    ],
+    "timeout": 0,
+    "spec": "./test/**/*.ts",
+    "recursive": true
+  },
   "author": "mapikit",
   "license": "GPL-3.0",
   "bugs": {
     "url": "https://github.com/mapikit/meta-function-helper/issues"
   },
   "homepage": "https://github.com/mapikit/meta-function-helper",
   "dependencies": {
+    "@meta-system/object-definition": "^1.0.6",
     "chalk": "^4.1.0",
+    "eslint": "^8.2.0",
```

Comment:
Good to see you added linting. In the future we should probably avoid creating any packages without it, no matter how simple they are.

That being said, I believe eslint should be moved to devDependencies.